<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QDialog" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Refbox Connection</string>
   </property>
   <widget class="QLineEdit" name="txt_port">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_fps">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>34</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p FPS</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_teamA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Team A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_teamB">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Team B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_teamB_robots">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0 robots</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_teamA_robots">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0 robots</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>21</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Reason for game stop</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pushing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Repair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ball Holding</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Long Dribble</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2 Robots</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Illegal Defense</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Illegal Attack</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>21</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Visualization Options</string>
   </property>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Slideshow Mode
(place PNG files on 
&quot;slides&quot; folder)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_flip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip Sides</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fullscreen">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fullscreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_windowed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Windowed</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_teamA_loadLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Logo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_teamA_loadModel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_teamB_loadModel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_teamB_loadLogo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Logo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_popup">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Popup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Show Message To Audience</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_msg">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
